docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst

```diff
@@ -0,0 +1,456 @@
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+1. This model will assume the existence of a "recovered" cause model state in an attempt to be consistent with the GBD assumption that no morbidity due to stomach cancer occurs more than ten years past incidence of the *clinical* phase of stomach cancer. The assumption also asserts that there is no recurrance of stomach cancer.
+
+2. This model assumes that the GBD incidence rate corresponds to the incidence of asymptomatic stomach cancer rather than *clinically detected* stomach cancer arising from symptomatic presentation at the doctor's office, which is a mix of pre-clinical (detection of pre-clinical stomach cancer from other non-stomach cancer related reasons) and clinical detections. This assumption has a few notable downstream limitations, including:
+
+	- simulation incidence of *clinical* stomach cancer will lag slightly behind forecasted incidence of stomach cancer due to the mean sojourn time period delay
+  - assume a short mean sojourn time 
+
+.. todo::
+
+  think more about these assumptions in relation to the sojourn time
+
+3. The prevalence of preclinical/screen-detectable stomach cancer is assumed to be equal to prevalence of detected stomach cancer (GBD prevalence of stomach cancer) scaled to the ratio of duration spent in the preclinical/screen-detectable state (mean sojourn time) and the clinical state (average survival time). This method relies on the assumption that GBD prevalence of stomach cancer represents clinical stomach cancers; this may be a reasonable assumption for China given that they do not have an aggressive screening program. 
+
+
+Cause Model Diagram
++++++++++++++++++++
+
+.. image:: cause_model_diagram.svg
+
+State and Transition Data Tables
+++++++++++++++++++++++++++++++++
+
+.. list-table:: State Definitions
+   :widths: 5 5 20
+   :header-rows: 1
+
+   * - State
+     - State Name
+     - Definition
+   * - S
+     - Susceptible
+     - Without cancer condition (may have pre-cancer states)
+   * - PC
+     - Pre-clinical asymptomatic cancer, endoscopy detectable  
+     - With asymptomatic condition, detectable through endoscopy screening, will progress to clinical symptomatic phase
+   * - C
+     - Clinical stomach cancer
+     - With symptomatic condition
+   * - R
+     - Recovered
+     - Without condition; not susceptible
+
+.. list-table:: States Data
+   :widths: 20 25 30 30
+   :header-rows: 1
+   
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - S
+     - prevalence
+     - 1 - prevalence_c414 x MST/AST - prevalence_c414
+     - Note: this assumes no initial prevalence in R state
+   * - S
+     - birth prevalence
+     - 0
+     - 
+   * - S
+     - excess mortality rate
+     - 0
+     - 
+   * - S
+     - disabilty weights
+     - 0
+     -
+   * - PC
+     - prevalence
+     - prevalence_c414 x MST/AST
```

Comment:
can you think this through with Yongquan?  I think this approach might have been part of the problem we have been working on in the cervical cancer model.

Reply:
@yongqx let's think through this in co-workign now?

Reply:
@Ninicorn Slack me your time preference!

Reply:
after co-working discussion with Yongquan and Ali, we decided that Ali will run this equation through the nano-sim and see if we should update this equation to prevalence (PC state) = incidence (i_c414 at age current age +MST) * duration (MST)